lib/services/in-memory-db.service.ts

```diff
@@ -67,7 +68,29 @@ export class InMemoryDBService<T extends InMemoryDBEntity> {
   }
 
   /**
-   * Update a record in the `recordMap` of type `T` using the supplied record.
+   * Add the supplied `records` partials array to in-memory data store of records.
+   * Get the `id` of the record by getting the next available `id` value.
+   * Returns a sequential array of the newly generated `ids`.
+   * @param records any array of partial records of type `T` to create
+   */
+  public createMany(records: Array<Partial<T>>): number[] {
```

Comment:
Would it be a bit easier on the calling method to return `T[]` instead of a `number[]`? I am just thinking it would be difficult to figure out which number returned from this method belongs to which input record.

Reply:
Yes I waffled back and forth on this one. I think I agree. I will refactor to return array of the full objects. Folks can always .map on the other side if they just care about IDs.